recipes/mfast/all/conanfile.py

```diff
@@ -15,13 +15,13 @@
 
 class mFASTConan(ConanFile):
     name = "mfast"
-    license = "LGPL-3.0"
-    url = "https://github.com/conan-io/conan-center-index"
-    homepage = "https://objectcomputing.com/"
     description = (
         "mFAST is a high performance C++ encoding/decoding library for FAST "
         "(FIX Adapted for STreaming) protocol"
     )
+    license = "LGPL-3.0"
```

Comment:
Is this license wrong? https://github.com/objectcomputing/mFAST/blob/master/licence.txt